prog/vnet/cert_nexus.rb

```diff
@@ -127,6 +138,13 @@ def before_run
     pop "certificate revoked and destroyed"
   end
 
+  def update_stack_restart_counter
+    current_frame = strand.stack.first
+    current_frame[:restarted] += 1
```

Comment:
why a frame variable rather than something in an entity? Is this strand not attached to any entity?

Reply:
It feels like the counter is very specific to the strand rather than entity. I didn't want to make a modification to the model to be honest.

Reply:
alright, I think that makes sense. So you don't think it fits with an existing data structure and doesn't justify a new one (not enough fields on the stack that are interesting enough for a schema?)